README.md

```diff
@@ -101,18 +96,18 @@ The standalone mode which has a large header with logo and a footer, can be enab
 You can run the unit tests by executing:
 
 ```bash
-pnpm test
+pnpm test:unit
 ```
 
 To run the e2e tests, run:
 
 ```bash
-pnpm test:e2e
+pnpm test:playwright
 ```
 
 You might have to run `npx playwright install` to get the browsers installed if e2e tests fail.
 
-When writing e2e tests, you can also use `pnpm generate-e2e-tests` to generate tests and test selectors.
+When writing e2e tests, you can also use `pnpm test:playwright:gen` to generate tests and test selectors.
```

Comment:
Let's delete this section altogether and link to the TESTING_GUIDELINES document so we don't duplicate this information.

Some of it is out of date (for example, the `npm playwright install` advice is generally going to be unnecessary for most people who will just be running things in docker anyway).

Reply:
Great advice! Implemented in 11ba387.